api/src/main/java/io/strimzi/api/kafka/model/common/authentication/KafkaClientAuthenticationOAuth.java

```diff
@@ -80,6 +81,16 @@ public void setClientId(String clientId) {
         this.clientId = clientId;
     }
 
+    @Description("OAuth grant type to use when authenticating against the authorization server. This value defaults to `client_credentials` when `clientId` and `clientSecret` are specified.")
```

Comment:
Is the default of `client_credentials` set within the Strimzi oauth library? As I don't see it being set anywhere in this PR

Reply:
Actually there is another case when this applies. What about the following description:

```suggestion
    @Description("A custom OAuth grant type to use when authenticating against the authorization server with `clientId` and one of `clientSecret` or `clientAssertion`. If not set, the value defaults to `client_credentials` in these cases. This is optional configuration, only used with custom authorization server implementations.")
```